airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -1662,11 +1662,8 @@ def _cleanse_time_partitioning(destination_dataset_table, time_partitioning_in):
 
     time_partitioning_out = {}
     if destination_dataset_table and '$' in destination_dataset_table:
-        if time_partitioning_in.get('field'):
-            raise ValueError(
-                "Cannot specify field partition and partition name"
-                "(dataset.table$partition) at the same time")
-        time_partitioning_out['type'] = 'DAY'
+        if not time_partitioning_in.get('field') and not time_partitioning_in.get("type"):
```

Comment:
I don't think we need this check. If `$` is present in table name, it should be partitioned by `DAY` as I guess "The only type supported is DAY, which will generate one partition per day."

Reply:
Also update this https://github.com/apache/incubator-airflow/blob/7d60d2686d40342255a5e326f31d3cc42c72a68b/airflow/contrib/hooks/bigquery_hook.py#L566

Reply:
you right, 5 sec